components/channel_header/components/header_icon_wrapper.tsx

```diff
@@ -105,6 +105,11 @@ const HeaderIconWrapper: React.FC<Props> = (props: Props) => {
         );
     }
 
+    let ariaLabelText;
```

Comment:
Am I missing something, or there are still cases where the aria label text is not defined? (if no `ariaLabel`, and not `tooltipText` or `tooltipText` not being a string)